values/tags.yaml.gotmpl

```diff
@@ -12,3 +12,10 @@ registry:
   credentials:
     username: {{ requiredEnv "REGISTRY_USERNAME" }}
     password: {{ requiredEnv "REGISTRY_PASSWORD" }}
+
+vaultPlugin:
+  tag: {{ env "VAULT_PLUGIN_TAG" | default "v0.0.5" }}
+  sha256: {{ env "VAULT_PLUGIN_SHA256SUM" | default "5d63d9891463c8b7dc281759c105b45835bc8e91cde019a9bde74d858f795740" }}
```

Comment:
This value should be downloaded from the repo assets, instead of being set as ENV var

Reply:
There is no way to link the initContainer and the initialization of the plugin when using the Vault operator. We could evntually download the SHA256 locally then pass as an env var but I find it less clean